charts/akeyless-api-gateway/values.yaml

```diff
@@ -18,6 +18,9 @@ initContainer:
     pullPolicy: IfNotPresent
 
 cache:
+  tls:
+    autoGenerated: false
+    secretName: ""
```

Comment:
remove secretName - for autoGenerated please generate a name...
please add existingSecretName

Reply:
secretName is existingSecretName, will rename